rocket.chat/rocket.chat-server/mac-osx.md

```diff
@@ -7,15 +7,10 @@ You can set up and run a Rocket.Chat development environment on your macOS syste
 
 ## Apple Silicon
 
-{% hint style="info" %}
```

Comment:
This info was added because of a popular issue some people faced and reported in the `support` and `GSoC` channel>
The hint helps in clarifying that.